assets/queries/cloudFormation/ec2_network_acl_duplicate_rule/query.rego

```diff
@@ -34,9 +34,21 @@ getRef(obj) = obj.Ref {
 }
 
 getTraffic(entry) = "egress" {
-	object.get(entry.Properties, "Egress", "undefined") == true
+	entry.Properties.Egress == true
```

Comment:
Is really necessary to check with and without `lower`? I think only checking with `lower` should be fine, no?

Reply:
Using lower only would cause problems when the property came as a boolean, however, Rogério suggested a solution that minimizes the number of elses in that method.